modules/nlu/src/views/lite/intentEditor/slots/Slots.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import _ from 'lodash'
+
+import SlotModal from './SlotModal'
+
+import style from '../style.scss'
+import SlotItem from './SlotItem'
+import { NonIdealState, Button } from '@blueprintjs/core'
+
+export default class Slots extends React.Component {
+  state = {
+    slotModalVisible: false,
+    editingSlotIdx: null
+  }
+
+  getSlots = () => this.props.slots
```

Comment:
Not sure of the value added with that shortcut, maybe it should return an empty array if undefined^

Reply:
That's some old code that will be trashed soon. So won't fix.